### PR TITLE
VB-312: add posibility to delete attached photo

### PR DESCRIPTION
**Pull request by voonya**
image

### CODE REVIEW

**Comment by voonya** [comment]
Dont merge now, checking for bugs

File: packages/frontend/components/save-post/image-input.tsx [hunk 1]
```diff
@@ -112,6 +117,15 @@ function ImageInput({ images, setImages }: ImageInputProps) {
           imagePreviews.map((item, indx) => (
             <div key={indx} css={styles.imgWrapper}>
               <Image objectFit="cover" layout="fill" src={item} />
+              <IconButton
+                cssExtend={styles.deleteImageBtn}
+                ariaLabel="close"
```
Comment (Nightlngale) [inline packages/frontend/components/save-post/image-input.tsx:122]: please add localisation

**Review by Nightlngale — APPROVED** [review]

File: packages/frontend/components/save-post/image-input.tsx [hunk 2]
```diff
@@ -78,6 +78,11 @@ function ImageInput({ images, setImages }: ImageInputProps) {
     setCurrentImage(undefined);
   };
 
+  const onImageDelete = (img: string) => {
+    setImagePreviews(imagePreviews.filter((item) => item !== img));
```
Comment (dannleed) [inline packages/frontend/components/save-post/image-input.tsx:82]: do we handle image deleting from server if it was already uploaded to storage?

**Review by YevheniiBieliaiev — APPROVED** [review]

File: packages/frontend/components/save-post/image-input.tsx (hunk 2 above)
Comment (voonya) [reply packages/frontend/components/save-post/image-input.tsx:82]: yep

**Pull request merged**